be/src/olap/rowset/segment_v2/inverted_index/query/phrase_query.cpp

```diff
@@ -73,14 +172,33 @@ void PhraseQuery::add(const std::wstring& field_name, const std::vector<std::str
     }
```

Comment:
no _matcher for terms.size() == 1 ?

Reply:
Matcher handles position information, terms.size() == 1 is handled in other processes